app/controllers/users/invitations_controller.rb

```diff
@@ -14,10 +14,15 @@ def new
     end
 
     def create
-      user_for_authorisation = User.new(invite_params)
+      user_for_authorisation
       authorize user_for_authorisation
-
-      super
+      if user_for_authorisation.deleted?
+        flash[:alert] = t('devise.invitations.user_exists', email: Settings.mail.tech_support)
+        @user = user_for_authorisation
+        render :new
```

Comment:
OK, I think I understand now, because they could just choose a different e-mail and create the user again. But then it more seems like a validation concern, which could be displayed as an error on the `email` field? Probably something to ask Adrian about, from product perspective.